ShopperInsights/src/androidTest/java/com/braintreepayments/api/shopperinsights/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.braintreepayments.api.shopperinsights
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
Let's rename this to the actual class or remove if we aren't going to have instrumentation tests here (sometimes they live in `Demo` module integration tests only